src/views/TradingCompetition/components/PrizesInfo/PrizesGrid/index.tsx

```diff
@@ -1,22 +1,178 @@
 import styled from 'styled-components'
-import HeadingRow from './HeadingRow'
-
-const Wrapper = styled.div`
-  display: grid;
-  grid-gap: 8px;
-  grid-template-columns: repeat(5, auto);
-  grid-template-rows: repeat(6, 1fr);
-  margin-bottom: 24px;
+import {
+  BlockIcon,
+  Box,
+  CheckmarkCircleIcon,
+  Flex,
+  MedalBronzeIcon,
+  MedalGoldIcon,
+  MedalPurpleIcon,
+  MedalSilverIcon,
+  MedalTealIcon,
+  CrownIcon,
+  Tab,
+  TabMenu,
+  Text,
+  TeamPlayerIcon,
+  TrophyGoldIcon,
+} from '@pancakeswap-libs/uikit'
+import useI18n from 'hooks/useI18n'
+import easterPrizes, { Tiers, Rank } from 'config/constants/trading-competition/easter'
+
+const COLOR_GOLD = '#FFBF33'
+const COLOR_SILVER = '#C1C1C1'
+const COLOR_BRONZE = '#E79559'
+const COLOR_PURPLE = '#A57CFD'
+const COLOR_TEAL = '#4CD2DD'
+
+const tierStyleMap = {
+  [Tiers.GOLD]: {
+    icon: MedalGoldIcon,
+    label: {
+      id: 999,
+      fallback: 'Gold',
+    },
+    color: COLOR_GOLD,
+  },
+  [Tiers.SILVER]: {
+    icon: MedalSilverIcon,
+    label: {
+      id: 999,
+      fallback: 'Silver',
+    },
+    color: COLOR_SILVER,
+  },
+  [Tiers.BRONZE]: {
+    icon: MedalBronzeIcon,
+    label: {
+      id: 999,
+      fallback: 'Bronze',
+    },
+    color: COLOR_BRONZE,
+  },
+  [Tiers.SILVER]: {
+    icon: MedalSilverIcon,
+    label: {
+      id: 999,
+      fallback: 'Silver',
+    },
+    color: COLOR_SILVER,
+  },
+  [Tiers.PURPLE]: {
+    icon: MedalPurpleIcon,
+    label: {
+      id: 999,
+      fallback: 'Purple',
+    },
+    color: COLOR_PURPLE,
+  },
+  [Tiers.TEAL]: {
+    icon: MedalTealIcon,
+    label: {
+      id: 999,
+      fallback: 'Teal',
+    },
+    color: COLOR_TEAL,
+  },
+}
+
+const Td = styled.td`
+  border-bottom: 2px solid ${({ theme }) => theme.colors.borderColor};
+  padding: 8px;
+`
+const BoldTd = styled(Td)`
+  font-weight: 600;
+`
+
+const PrizeTable = styled.table`
+  width: 100%;
+
+  th,
+  td {
+    text-align: center;
+    vertical-align: middle;
+  }
+
+  & > thead th {
+    font-size: 12px;
+    padding: 16px;
+    text-transform: uppercase;
+  }
 `
 
+const getTotalAchievementPoints = (achievements: Rank['achievements']) => {
+  return Object.values(achievements).reduce((accum, achievement) => {
+    return achievement ? accum + achievement : accum
+  }, 0)
+}
+
 const PrizesGrid = () => {
+  const [tab, setTab] = useState(0)
+  const TranslateString = useI18n()
+  const rows = easterPrizes[tab + 1]
+
+  const handleItemClick = (index: number) => setTab(index)
+
   return (
-    <Wrapper>
-      <HeadingRow />
-      <span>Tabmenu</span>
-      <span>Table</span>
-    </Wrapper>
+    <Box pt="24px">
+      <TabMenu activeIndex={tab} onItemClick={handleItemClick}>
+        {Object.keys(easterPrizes).map((team) => {
+          return <Tab key={team}>{TranslateString(999, `#${team} Team`, { num: team })}</Tab>
+        })}
+      </TabMenu>
+      <Box minWidth="288px" overflowX="auto" maxWidth="100%">
+        <PrizeTable>
```

Comment:
I saw that your origin stuff was using grid. In this case I elected for a table because the component is tabular and there doesn't seem to be much difference in the mobile version.

Reply:
Yea I went with grid styles because the table html syntax just makes me prang out haha. Makes total sense for this to be a table though 👌